initial ssl_exporter implementation (#1692)
grafana/agent: docs/user/configuration/integrations/ssl-config.md

+++
title = "ssl_exporter_config"
+++

# ssl config

The `ssl` block configures the `ssl` integration,
which is an embedded version of
[`ssl_exporter`](https://github.com/ribbybibby/ssl_exporter). This allows collection of SSL certificate metrics from hosts and files.

## Quick configuration example

> To get started, define SSL targets in Grafana agent's integration block:

Reviewer: ```suggestion
Define SSL targets in Grafana Agent's integration block:
```

  [module: <string> | default = "tcp"]
```

## About ssl_exporter Modules

> Read more about the supported modules at [ribbybibby/ssl_exporter](https://github.com/ribbybibby/ssl_exporter#configuration)

Reviewer: ```suggestion
For more information on the supported modules, refer to [ribbybibby/ssl_exporter](https://github.com/ribbybibby/ssl_exporter#configuration)
```
